src/files-and-videos/files-page/FilesPage.jsx

```diff
@@ -90,7 +91,8 @@ const FilesPage = ({
     usageErrorMessages: errorMessages.usageMetrics,
     fileType: 'file',
   };
-  const maxFileSize = 20 * 1048576;
+  //const maxFileSize = 20 * 1048576;
```

Comment:
```suggestion
```
Please just delete this line altogether :)
You could also get rid of the `maxFileSize` variable and just use `UPLOAD_FILE_MAX_SIZE` throughout the file.